CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG for mcrouter
 
+## [v3.0.0] (2017-08-15)
+
+### Breaking changes:
+* Removed support for Ubuntu 14.04
+* Converted `node['folly']['version']` to `node['mcrouter']['commit_hash']` given that FB has a weird release process.
```

Comment:
@Brantone @mitch-roblox I'm just going to put back the attributes with the commit hashes and update the README clarifiying that if you want to update mcrouter version don't forget to check if `folly`, `double-conversion` and `wangle` commit hashes have changed.